### ISSUE TITLE
invalid memory address or nil pointer dereference running destroy cluster

### ISSUE THREAD

**Issue by coreydaley** [openshift/installer#1547]
```
$ ./bin/openshift-install version
./bin/openshift-install ee5d518c27c00d46a411725fd6e29e3f6ce7608a
built from commit ee5d518c27c00d46a411725fd6e29e3f6ce7608a
release image registry.svc.ci.openshift.org/openshift/origin-release:v4.0
```

```
$ ./bin/openshift-install destroy cluster --dir=cluster
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0xf909e2]

goroutine 1 [running]:
github.com/openshift/installer/pkg/asset.DeleteAssetFromDisk(0x5dd8cc0, 0xaaab7a0, 0x7ffed47e1678, 0x7, 0xaaab7a0, 0xc00083b201)
	/home/cdaley/go/src/github.com/openshift/installer/pkg/asset/asset.go:67 +0x182
github.com/openshift/installer/pkg/asset/store.(*storeImpl).Destroy(0xc00083b230, 0x5dbb4c0, 0xaaab7a0, 0x5dbb4c0, 0xaaab7a0)
	/home/cdaley/go/src/github.com/openshift/installer/pkg/asset/store/store.go:103 +0x400
main.runDestroyCmd(0x7ffed47e1678, 0x7, 0xc000a09f00, 0xc0008b3bc8)
	/home/cdaley/go/src/github.com/openshift/installer/cmd/openshift-install/destroy.go:60 +0x180
main.newDestroyClusterCmd.func1(0xc0009b1180, 0xc0009a2a10, 0x0, 0x1)
	/home/cdaley/go/src/github.com/openshift/installer/cmd/openshift-install/destroy.go:38 +0x73
github.com/openshift/installer/vendor/github.com/spf13/cobra.(*Command).execute(0xc0009b1180, 0xc0009a29e0, 0x1, 0x1, 0xc0009b1180, 0xc0009a29e0)
	/home/cdaley/go/src/github.com/openshift/installer/vendor/github.com/spf13/cobra/command.go:766 +0x2cc
github.com/openshift/installer/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0xc0009b0780, 0x8, 0x0, 0xc000a0e280)
	/home/cdaley/go/src/github.com/openshift/installer/vendor/github.com/spf13/cobra/command.go:852 +0x2fd
github.com/openshift/installer/vendor/github.com/spf13/cobra.(*Command).Execute(0xc0009b0780, 0xc0008b3ee0, 0x1)
	/home/cdaley/go/src/github.com/openshift/installer/vendor/github.com/spf13/cobra/command.go:800 +0x2b
main.installerMain()
	/home/cdaley/go/src/github.com/openshift/installer/cmd/openshift-install/main.go:57 +0x14f
main.main()
	/home/cdaley/go/src/github.com/openshift/installer/cmd/openshift-install/main.go:40 +0x99
```

**Comment by wking** [comment]
Fix discussion happening in #1548.

**Comment by coreydaley** [comment]
Awesome, thanks @wking
